js/upload-file/README.md

```diff
@@ -0,0 +1,13 @@
+# Upload a File
+
+Sample that uses fetch to upload a file. 
+
+## Dependencies
+None, just open in a browser
+
+## Instructions
+1. Open the upload.html file in a browser
+2. enter a valid access token
```

Comment:
If you don't have it already, you can request an access token by going in the 'Developer Playground' section of the User Profile in your account at App.Docugami.com. Note you need to be an administrator in the account to have access to Developer Playground

Reply:
I just put in a link to the help article which explains this in detail.